resources/css/bem/admin-contest.less

```diff
@@ -2,9 +2,38 @@
 // See the LICENCE file in the repository root for full licence text.
 
 .admin-contest {
-  &__meta-row {
-    margin-bottom: 5px;
-    color: @osu-colour-f1;
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+
+  &__description {
+    background-color: hsl(var(--hsl-b4));
+    font-size: @font-size--title-small;
+    padding: 10px;
+
+    @media @narrow-up {
+      padding: 20px;
+    }
+  }
+
+  &__meta-title {
+    .u-ellipsis-overflow();
```

Comment:
shouldn't be needed? is this related to the 160px? which label is too long? (and even then it should have tooltip at least?)